### DISCUSSION TITLE
오브젝트끼리의 충돌

### DISCUSSION THREAD

**Discussion by YNyum** [naverz/zepeto-studio-kor#805] [5. Q&A]
빌드 잇 패키지에서 불러온 오브젝트들이 collider를 가지고있음에도 충돌이 일어나지 않는데 이유를 알 수 있을까요?
캐릭터와는 충돌이 되지만 오브젝트들끼리 충돌이 안됩니다.
isTrigger해제 된 상태이며, Layer도 20~27사이에서 사용중입니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 빌드잇 패키지 오브젝트에 한정된 부분은 아니고
오브젝트끼리의 충돌을 인식하려면 어느 한쪽은 Rigidbody가 존재해야 합니다.
오브젝트에 Rigidbody를 추가해서 테스트 해 보세요. 

<img width="363" alt="스크린샷 2022-05-27 오전 10 49 44" src="https://user-images.githubusercontent.com/100118650/170612496-56424aa2-9111-43f3-86b6-c41977caae0e.png">
